test/task.test.ts

```
@@ -85,7 +86,10 @@ total-bounty-eth: add(
     });
 
     it("should parse rainlang to bytecode", async function () {
-        const viemClient = await createViemClient(1, [mockServer.url + "/rpc"]);
+        const viemClient = await createViemClient(
+            1,
+            new RpcState([{ url: mockServer.url + "/rpc" }]),
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)